test/graph_test/graph_test.cpp

```diff
@@ -157,6 +157,31 @@ void TestHelper::executeScript(const std::string& cypherScript, Connection& conn
         }
     }
 }
+void BaseGraphTest::createDB(uint64_t checkpointWaitTimeout) {
```

Comment:
Let's also try to have only 2 functions.

```
createDB(checkpointWaitTimeout)
createConn(checkpointWaitTimeout)
```
 If checkpointWaitTimeout = 0, the behaviour will fall back to createDBAndConn()

Reply:
ok